Testing/functional/tests/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
     "env": {
-        "cypress/globals": true
+        "cypress/globals": true,
+        "es2021": true
```

Comment:
What's this change for?

Reply:
That came from a commit from dev.  This PR should be updated to drop all the commits except the last one.